Meta/WPT.sh

```diff
@@ -183,12 +239,232 @@ update_wpt() {
     popd > /dev/null
 }
 
-execute_wpt() {
-    # Ensure open files limit is at least 1024, so the WPT runner does not run out of descriptors
-    if [ "$(ulimit -n)" -lt 1024 ]; then
-        ulimit -S -n 1024
+declare -a WPT_NS=()
+
+cleanup_run_infra() {
+  readarray -t NSS < <(ip netns list 2>/dev/null | grep -E '^wptns[0-9]+' | awk '{print $1}')
```

Comment:
Well one major issue I'd have with all this: are we leaving behind macOS support? Currently I'm using this script heavily to run WPT tests there. Pretty sure all this `ip netns` stuff is not going to cut it.

Reply:
This is impossible to do on macos, the script will just give up and run sequentially.
I've done some research, and short of running this all in a linux VM (which defeats the purpose) there's no way AFAICT.

Reply:
Well, there's _one_ way that I haven't tested, but I really don't want to do it, LD_PRELOAD/DYLD_INSERT_LIBRARIES a proxy for bind and connect which translates 127.0.0.1 to 127.0.0.N (but that still won't solve the issue with needing separate working dirs _and_ the same wpt dir structure without symlinks, no overlayfs on macos)

Reply:
Wait are you saying there's a fallback mode in the script so it still works on macOS? If so, carry on...

Reply:
Yes, it'll just work like it currently does if there's no `ip netns` or if it's "defective" (it makes a test ns)

Reply:
I don't think there is an equivalent to the `ip netns` command in MacOS - the version of `ip` I installed certainly didn't have it. Personally, I'd be fine with this feature not being supported on MacOS - to me the main use case of this feature is speeding up the WPT runner, which only runs on Linux at the moment.